crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -84,7 +82,16 @@ const FINAL_EPOCH_CHECKPOINT_INDEX: u64 = 0;
 const OVERRIDE_PROTOCOL_UPGRADE_BUFFER_STAKE_INDEX: u64 = 0;
 pub const EPOCH_DB_PREFIX: &str = "epoch_";
 
+// CertLockGuard and CertTxGuard are functionally identical right now, but we retain a distinction
+// anyway. If we need to support distributed object storage, having this distinction will be
+// useful, as we will most likely have to re-implement a retry / write-ahead-log at that point.
 pub struct CertLockGuard(MutexGuard);
+pub struct CertTxGuard(CertLockGuard);
+
+impl CertTxGuard {
+    pub fn release(self) {}
```

Comment:
What is the plan for this methods in the future? Are we going to delete them when we get a bit more time?